app/javascript/components/Editor.tsx

```diff
@@ -521,6 +550,20 @@ export default ({
               </TasksContext.Provider>
             }
           />
+          <RealtimeFeedbackModal
+            open={feedbackModalOpen}
+            onClose={hideFeedbackModal}
+            discussion={discussion}
+            mentoringStatus={mentoringStatus}
+            onSubmit={submit}
+            solution={solution}
+            track={track}
+            request={request}
+            submission={submission}
+            exercise={exercise}
+            trackObjectives={trackObjectives}
+            links={{ ...links, redirectToExerciseLink: redirectLink }}
```

Comment:
```suggestion
            links={{ ...links, redirectToExercise: redirectLink }}
```

General comment: I feel like if we have `links: { ... }` then the things inside `...` probably don't need the `link` suffix.

Reply:
I usually add it, if only one link is passed down. and I want to know it is a link